jupyter-notebooks/orders_basemaps/SDK_order_basemaps.ipynb

```diff
@@ -0,0 +1,403 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Order a Basemap Using the Orders API & SDK via AOI & Quad ID"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview ##\n",
+    "---\n",
+    "Download Basemap quads based on the mosaic name, merged and clipped to the area of interest.\n",
+    "\n",
+    "For example, from Explorer, I've defined my area of interest and downloaded that geoJSON.\n",
+    "\n",
+    "I've selected the basemap:\n",
+    "\n",
+    "global_monthly_2022_01_mosaic\n",
+    "\n",
+    "and downloaded my Area of Interest.\n",
+    "\n",
+    "To view the basemap, I clicked on the arrow-in-a-box icon to open that basemap in the Basemap viewer.\n",
+    "\n",
+    "To see all quads in my subject area, I uploaded my Area of Interest.\n",
+    "\n",
+    "326-1257,327-1257,328-1257,326-1256,327-1256,328-1256,326-1255,327-1255,328-1255\n"
```

Comment:
If you do the outline I mentioned above, you can detail each of these steps in the corresponding section in the notebook.

I also recommend referencing the image names as code, so they jump out:
`global_monthly_2022_01_mosaic` instead of "global_monthly_2022_01_mosaic"

This way, you don't have to add as many empty lines and it'll still be clear.